Firestore/core/src/util/log_stdio.cc

```diff
@@ -26,20 +27,20 @@ namespace firestore {
 namespace util {
 namespace {
 
-LogLevel g_log_level = kLogLevelNotice;
+std::atomic<LogLevel> g_log_level(kLogLevelNotice);
 
 }  // namespace
 
 void LogSetLevel(LogLevel level) {
-  g_log_level = level;
+  g_log_level.store(level);
```

Comment:
Any reason not to use the overloaded operators? From a glance, it seems like other than `s/LogLevel/std::atomic<LogLevel>/`, the code in this file could be left as it was.

Reply:
I chose to use the methods instead of the overloaded operators to highlight that they are atomic operations. I watched a CppCon video (I forget which one exactly) about atomics and the speaker recommended doing it this way. Since I hadn't come across any dissenting opinions, I just went with it :) I'm happy to do whatever feels more conventional in our code base. Please advise.

Reply:
As long as it is a deliberate choice, I think that's fine. Our existing usage somewhat favors the overloaded operators, but there's not much usage. Personally, I think classes should behave like built-in types syntax-wise where appropriate, but it's really a matter of personal preference.

Reply:
Done. I've changed the load()/store() calls to use the overloaded operators, since I am ambivalent towards the two options.